src/birthder.js

```diff
@@ -128,10 +129,30 @@
    * @returns {string} - Image URL.
    */
   function selectTenorImageUrl (response) {
-    let items = response.results
-    let randomItems = items[Math.floor(Math.random() * items.length)]
+    const items = response.results
 
-    return randomItems.media[0].gif.url
+    const filteredItems = items.filter(item => {
+      const re = /^(https?[:]\/\/media.tenor.com\/images\/([0-9a-z]+)\/tenor.gif)$/i
+      const match = item.media[0].gif.url.match(re)
+
+      if (!match) {
+        return false
+      }
+
+      if (TENOR_BLACKLIST.includes(match[2])) {
+        console.log('Tenor returned the image from blacklist. Retrying...')
```

Comment:
Let's amend the record a bit.
`Tenor returned the image from the blacklist specified via TENOR_BLACKLIST. Retrying...`